### DISCUSSION TITLE
Duda IN-RECTANGLE e IN-CIRCLE

### DISCUSSION THREAD

**Discussion by ncruzm** [IIC2133-PUC/2024-2#146] [Tarea 2] [answered]
Holaa, una pregunta en estos dos eventos podria ocurrir lo mismo que en los anteriores de que dos pingüinos tengan la misma llave en este caso la misma coordenada? O asumimos que todas las coordenadas serán distintas?

**Comment by Segismundios** [comment, marked as answer]
Hola @ncruzm. Puedes asumir que todas las coordenadas serán distintas. 
Saludos!
